Gruntfile.js

```diff
@@ -120,10 +120,31 @@ module.exports = (grunt) => {
         }
     });
 
+    grunt.loadNpmTasks('grunt-webext-builder');
     grunt.initConfig({
         manifest: grunt.file.readJSON('manifest.json'),
-        cwsOptions: grunt.file.readJSON('cws.json'),
         webpack: webpackConfigs,
+        webext_builder: {
+            chrome: {
+                privateKey: ".private.pem",
+                targets: [
+                  "chrome-crx"
+                ],
+                files: {
+                  "dist/":["build/chrome"]
+                }
+            },
+            firefox: {
+                jwtIssuer: process.env.WEB_EXT_API_KEY,
```

Comment:
Could you also add an option of using some configuration files (like what I've called `amo.json`)?

Reply:
I'm working on a way to make Grunt use either the environmental variable (with precedence) or `amo.json`/`cws.json` so you shouldn't need to change a lot your current workflow. This should resolve the issues you've mentioned in your last 2 comments. Never the less, I'm not sure about 1 thing:

It seems that currently, it is possible to pack a development version and a stable version of the extension. I'm not sure why you've made a distinction between the two and how exactly it works with `Gruntfile.js`. As for my PR, I'm not sure whether it's possible to do so with [grunt-webext-builder](https://github.com/MobDev-Hobby/grunt-webext-builder) . What do you think?